/usr/local/lib/python3.6/dist-packages/pytorch_lightning/utilities/distributed.py:50: UserWarning: You have set progress_bar_refresh_rate < 20 on Google Colab. This may crash. Consider using progress_bar_refresh_rate >= 20 in Trainer.
  warnings.warn(*args, **kwargs)
GPU available: True, used: True
TPU available: None, using: 0 TPU cores
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name | Type   | Params
--------------------------------
0 | unet | UNet   | 8.6 M 
1 | loss | L1Loss | 0     
--------------------------------
8.6 M     Trainable params
0         Non-trainable params
8.6 M     Total params






[master 0324eba] Added video loggining
 23 files changed, 1405 insertions(+), 5 deletions(-)
 create mode 100644 wandb/run-20210202_194211-23dlcc13/files/config.yaml
 create mode 100644 wandb/run-20210202_194211-23dlcc13/files/media/videos/Car_0_e028b3b1.mp4
 create mode 100644 wandb/run-20210202_194211-23dlcc13/files/media/videos/Car_1_ce2beef2.mp4
 create mode 100644 wandb/run-20210202_194211-23dlcc13/files/media/videos/Car_2_2a08ae92.mp4
 create mode 100644 wandb/run-20210202_194211-23dlcc13/files/media/videos/Car_3_ad61fd23.mp4
 create mode 100644 wandb/run-20210202_194211-23dlcc13/files/media/videos/Car_4_401d3771.mp4
 create mode 100644 wandb/run-20210202_194211-23dlcc13/files/output.log
 create mode 100644 wandb/run-20210202_194211-23dlcc13/files/requirements.txt
 create mode 100644 wandb/run-20210202_194211-23dlcc13/files/wandb-metadata.json
 create mode 100644 wandb/run-20210202_194211-23dlcc13/files/wandb-summary.json
 create mode 100644 wandb/run-20210202_194211-23dlcc13/logs/debug-internal.log
 create mode 100644 wandb/run-20210202_194211-23dlcc13/logs/debug.log
 create mode 100644 wandb/run-20210202_194211-23dlcc13/run-23dlcc13.wandb
To https://github.com/nicolas-dufour/instance-video-colorization.git
 ! [rejected]        master -> master (fetch first)
error: failed to push some refs to 'https://nicolas-dufour:8dS29ta%26v%26N%2A77d@github.com/nicolas-dufour/instance-video-colorization.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
